card/spec/lib/card/format/render_spec.rb

```diff
@@ -0,0 +1,19 @@
+describe Card::Format::Render do
+  describe "view cache" do
+    before { Cardio.config.view_cache = true }
+
+    let(:cache_key) do
+      "a-Card::Format::HtmlFormat-normal-home_view:content;"\
+      "nest_name:A;nest_syntax:A|content;view:contentcontent:show"
+    end
+
+    subject { Card::Cache[Card::View] }
+
+    it "can be changed with nest option" do
+      is_expected.to receive(:fetch).with cache_key
+      render_content "{{A|content}}"
+      is_expected.not_to receive(:fetch)
+      render_content "{{A|cache:never}}"
```

Comment:
Actually, I think the current code affects all three of these
1. the home view
2. views nested by the home view (it's listed as an "heir" option in options.rb)
3. other views navigated to by the user within the same slot, eg by clicking on edit

As a side note, I'm wondering if we should do something about all the content overridden by structure and structure overridden by code.  it's quite confusing.